### ISSUE TITLE
Bug Report: Bf's sing animation replaced by his idle animation.

### ISSUE THREAD

**Issue by VitaminaCaroll** [KadeArchive/Kade-Engine#2042]
So i was charting a song in kade engine, but when i play it and its bf's turn, sometimes he plays his idle animation instead of his regular singing animation ( as if the idle anim was a sing anim), I've tried to rechart it, it works there, but if i save the json and load it again, the bugs happens reappears. If someone knows how to fix this please let me know.

Im on windows 10, on kade engine 1.5.4

**Comment by Trayfellow** [comment]
yeah bf's animations end too early in kade engine 1.5.4, use an older version like 1.5.3 or 1.5.2 or just update to the latest one i guess

**Comment by VitaminaCaroll** [comment]
> yeah bf's animations end too early in kade engine 1.5.4, use an older version like 1.5.3 or 1.5.2 or just update to the latest one i guess

yeah I actually got the latest version to see if its fixed, but just got lazy to convert my mod (wich is like 70% complete), but now im gon do it, thanks for the help
